Dockerfile.old_test_suite

```diff
@@ -0,0 +1,62 @@
+FROM openbuildservice/base
+ARG CONTAINER_USERID
+
+# FIXME: https://bugzilla.opensuse.org/show_bug.cgi?id=957818
+RUN rm -rf /var/cache/zypp/*
+
+# Install requirements for the frontend
+RUN zypper -n install --no-recommends --replacefiles \
+  # as search daemon
+  sphinx \
+  # for testing javascript driven pages
+  phantomjs \
+  nodejs \
+  # for accessing the database
+  mariadb-client \
+  # for rspec (it does something with git)
+  git-core \
+  # dependencies for building gems
+  ruby2.4-devel cyrus-sasl-devel openldap2-devel libxml2-devel zlib-devel libxslt-devel \
+  # gems we don't want to build
+  ruby2.4-rubygem-mysql2 \
+  # gems we use that are not in our bundle
+  ruby2.4-rubygem-bundler ruby2.4-rubygem-thor-0_19 ruby2.4-rubygem-foreman
+
+# Install memcached
+RUN zypper -n install --no-recommends --replacefiles memcached
+
+# Install requirements for the backend
+RUN zypper -n install --no-recommends --replacefiles \
+  obs-server obs-signd \
+  obs-service-download_src_package obs-service-download_files \
+  obs-service-download_url \
+  obs-service-format_spec_file obs-service-kiwi_import \
+  perl-Devel-Cover \
+  osc
```

Comment:
I must say I'm not too fond of just copying every `RUN` into this Dockerfile. We are going to live with this setup for quite some time (until we switched everything to rspec) and this means there are always two places I have to adopt for software/configuration changes. 

How about moving this (back) into a script that you can call with arguments. Something like

```
RUN contrib/docker-bootstrap.sh backend
```

to do the backend part

```
RUN contrib/docker-bootstrap.sh signer
```

to do the signer part etc.

The docker images then can call this script in any sequence they need and we have only one place we need to adopt.

Reply:
Wouldn't the docker way be to have a docker file that does the common task and the use that image in any other docker files that's needs these packages?